Source/WebCore/platform/LocalizedStrings.cpp

```diff
@@ -107,11 +113,7 @@ RetainPtr<CFStringRef> copyLocalizedString(CFStringRef key)
 
     static CFStringRef notFound = CFSTR("localized string not found");
 
-#if PLATFORM(WIN)
-    CFBundleRef bundle = webKitBundle();
-#else
     CFBundleRef bundle = webCoreBundle();
```

Comment:
It doesn't look like you need the temporary here.

Reply:
Fixed.